src/commands/github_post_comment.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+description: Post a comment under a github PR.
+
+parameters:
+  comment:
+    type: string
+    default: "/review"
+    description: Comment to post.
+
+steps:
+  - run:
+      name: Post comment under PR
+      command: >-
+          PR_NUMBER=$(echo "${CIRCLE_PULL_REQUEST}" | awk -F '/' '{print $NF}')
+          BASE_URL="https://api.github.com/repos/noosenergy/${CIRCLE_PROJECT_REPONAME}"
+
+          set -- "$@" -X POST
+
+          set -- "$@" -H "Accept: application/vnd.github+json"
+
+          set -- "$@" -H "Authorization: Bearer ${GITHUB_TOKEN}"
+
+          set -- "$@" -H "X-GitHub-Api-Version: 2022-11-28"
```

Comment:
🐼 

Maybe as suggested by the AI PR agent, what about being able to pass the Github API version as a parameter?